src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsReconciliationService.kt

```diff
@@ -220,7 +221,7 @@ data class IncidentReportDetail(
   val type: String? = null,
   val status: String? = null,
   val reportedBy: String,
-  val reportedDateTime: String,
+  val reportedDateTime: LocalDateTime,
```

Comment:
was better to grab latest incidents code as they've done the `LocalDateTime` thing at the same time as us, so required less conversion